docs/examples/ex_extract.rst

```diff
@@ -0,0 +1,458 @@
+.. _ex_extract:
+
+Extract Observational-Style Data
+================================
+
+Comparison of model and Instrument data is supported in pysatModels, in part,
+by enabling the extraction (or interpolation) of model output onto the same
+locations as an observation-style data set. One common example is
+'flying' a satellite through a model. The satellite locations are used
+to extract relevant model data enabling direct comparison of observed and
+modeled values.
+
+
+Regular Grid Models
+-------------------
+
+:py:func:`pysatModels.utils.extract.extract_modeled_observations` supports
+extracting values from models on a regular grid. The function can linearly
+inteprolate model values onto instrument locations or use the nearest modeled
+location. Uses :py:func:`scipy.interpolate.interpn` as the underlying
+interpolation function.
+
+Leaving description of @aburrell's function to @aburrell.
+
+
+:py:func:`pysatModels.utils.extract.instrument_view_through_model` supports
+interpolating values from regular grid models onto Instrument locations using
+:py:func:`scipy.interpolate.RegularGridInterpolator`. Consider the following
+example that interpolates model data onto a satellite data set using
+pysat testing data sets.
+
+.. code:: python
+
+   import pysat
+   import pysatModels
+
+   # Load simulated satellite Instrument data set
+   inst = pysat.Instrument('pysat', 'testing', max_latitude=45.)
+   inst.load(2009, 1)
+
+   # Load simulated regular-grid model Instrument
+   model = pysat.Instrument('pysat', 'testmodel')
+   model.load(2009, 1)
+
+Looking at the loaded `model.data` we can see that the model is indeed regular.
```

Comment:
There is an inconsistent use of the `:py:thing:` style in this file.  I fixed some sections to give you more examples, but this whole file needs to be scrubbed.